src/Core/ServiceCollectionExtensions.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
What is the change here? Encoding?

Reply:
Yea, not sure why the hell VS did that. .editorconfig is clearly saying utf-8 but it had saved as Western European something.. sigh. Fixed.